### PR TITLE
bug/BL-532-ux-redesign-the-application-forms

### PR DESCRIPTION
**Pull request by ezra-casas**
## Description

Cleaned up the UI of the Mentor and Mentee signup forms. Previous UI was overlapping other elements on the page. New UI Layout is more user friendly and digestible.

Fixes # (issue)
Overlapping of labels and their inputs.

## Loom Video

(https://www.loom.com/share/4c91ce23322445a28ce97ce9e9ac4a39)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 1]
```diff
@@ -70,394 +71,414 @@ const Mentee = ({ dispatch, error, successPage }) => {
...
+                      name="first_name"
+                      rules={[
+                        {
+                          required: true,
+                          message: 'First name is required.',
+                        },
+                      ]}
+                      value={formValues.first_name}
+                      onChange={handleChange}
+                      style={{ margin: '.5rem 1rem .5rem 0', gap: '0rem' }}
```
[the remark this answers is not in the document]
Comment (farhaannishtar) [reply src/components/pages/RoleSignup/Applications/Mentee.js:109]: Well done. Everything looks great to me. The margin / padding / border specifications in your CSS files looks great as evident in the rendering of the application forms. Good move in switching to rems.

**Review by ethanrmiles — APPROVED** [review]
New UI looks awesome, and such a good use of the Ant design library API!

**Review by farhaannishtar — APPROVED** [review]

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 2]
```diff
@@ -70,394 +71,414 @@ const Mentee = ({ dispatch, error, successPage }) => {
...
-                        value="low_income"
-                        onChange={e => handleChange(e, 'checkbox')}
-                        style={{ margin: '.2rem', width: '100%' }}
+                      <Checkbox.Group
+                        style={{
+                          display: 'flex',
+                          flexFlow: 'column',
+                          width: 'auto',
+                          gap: '.5rem',
+                          // margin: '0rem 1rem 1rem 1.5rem',
```
Comment (ryan-hamblin) [inline src/components/pages/RoleSignup/Applications/Mentee.js:270]: Please remove any instances of commented out code.

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 3]
```diff
@@ -70,394 +71,414 @@ const Mentee = ({ dispatch, error, successPage }) => {
...
-                    onChange={handleChange}
-                    style={{ margin: '0 1rem .5rem 1.5rem' }}
-                  >
-                    <Input.TextArea placeholder="Your answer" />
-                  </Form.Item>
-                </Col>
-              </Row>
+                  <Col md={20}>
+                    <Form.Item
+                      // className="menteeConvictions"
```
Comment (ryan-hamblin) [inline src/components/pages/RoleSignup/Applications/Mentee.js:301]: Please remove.

File: src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css [hunk 4]
```diff
@@ -1,57 +1,29 @@
...
 
-.locationUS{
-  display: flex;
-  flex-direction: column;
 }
-#mentorSubmitButton{
-  background-color: #003D71;
-  color: #FFF;
-  margin-bottom: 2vh;
+.em{
```
Comment (ryan-hamblin) [inline src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css:20]: please put a space between the .em and "{".

File: src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css [hunk 5]
```diff
@@ -1,57 +1,29 @@
...
+  padding: 2rem;
 }
 
-.applicationForm{
-  border-radius:'4px';
-  border:solid 1px rgba(128, 128, 128, 0.342);
-  display: flex;
-  
-}
+.col-container{
```
Comment (ryan-hamblin) [inline src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css:16]: please put a space between the .col-container and "{".

File: src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css [hunk 6]
```diff
@@ -1,57 +1,29 @@
-.mentorApplication{
+.container{
   display: flex;
-  padding:2vh 6vw;
   justify-content: center;
 }
 
 .mentorTitle{
```
Comment (ryan-hamblin) [inline src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css:6]: please put a space between the .mentorTitle and "{".

**Review by ryan-hamblin — CHANGES_REQUESTED** [review]
The only critiques I have are simple styling preferences and omissions of commented out code. Great work. I love that you're sticking with the standard antD lib... even when you are forced to use an `important` tag you're getting it done.

**Review by jettyblue — APPROVED** [review]
Well done making the form look more presentable and having the text stretch out the whole line instead of broken up into multiple lines before the input sections. Besides the quick removal of commented out code requested by Ryan, this all looks great!

File: src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css (hunk 4 above)
Comment (miguelaledesma) [reply src/components/pages/RoleSignup/Applications/Styles/mentorApplication.css]: this has been resolved

**Review by farhaannishtar — DISMISSED** [review]

**Review by jettyblue — APPROVED** [review]
Everything looks great, well done!

**Comment by woodbenwood** [comment]
Thorough editing and cleanup, hard to find flaws with so much attn to detail

**Comment by woodbenwood** [comment]
> Thorough editing and cleanup, hard to find flaws with so much attn to detail

**Comment by woodbenwood** [comment]
Didn't intend to close, and feels weird I have that power. Testing Comments vs. Reviewing.

**Review by wenjuncai97 — APPROVED** [review]
Good job redesigning the application form! The form looks a lot better from the user's perspective, and it makes our forms look more legitimate.

**Review by MelodyLayne — APPROVED** [review]

**Review by zdravoj — APPROVED** [review]
You've done a lot of cleanup work and it's paid off. The form looks great and I'm glad it's compatible with mobile. Nice work!

**Pull request merged**